vscripts/sh_onboarding.gnut

```diff
@@ -513,6 +513,11 @@ void function OnPlayerKilled( entity victim, entity attacker, var damageInfo )
 		attacker.SetPlayerNetInt( "kills", attacker.GetPlayerNetInt( "kills" ) + 1 )
 
 	Remote_CallFunction_NonReplay( victim, "ServerCallback_YouDied", attacker, GetHealthFrac( victim ), DamageInfo_GetDamageSourceIdentifier( damageInfo ), DamageInfo_GetDamage( damageInfo ) )
+
+	victim.SetPredictionEnabled( false )
```

Comment:
You never re-enable prediction.
Afaik you will manually have to re-enable it and it will cause a ton of issues.

Reply:
For that matter, it seems to be automatically turned on in the native code
(This code is taken from Titanfall2 Northstar, and the code there says so too)
![image](https://user-images.githubusercontent.com/90076182/196061592-ff2da6b0-0fdd-42e8-81b6-eee77fb0dae2.png)

Reply:
In fact, even on r5r, there was no discrepancy in movement after respawning

Reply:
https://user-images.githubusercontent.com/90076182/196157582-6afb0e3c-0f5c-491a-a15c-5f617e3a59aa.mp4

I am aware of the problem that if prediction is not enabled, the animations will be jerky and bullets will not be visible
But that is not a problem because it is automatically enabled in the native code at respawn